schema/cpacs_schema.xsd

```diff
@@ -11330,36 +11341,93 @@ jonas.jepsen@dlr.de
 				<xsd:sequence>
 					<xsd:element name="tool" type="toolType">
 						<xsd:annotation>
-							<xsd:documentation>Tool identification</xsd:documentation>
+							<xsd:documentation>
```

Comment:
You should not commit unnecessary changes such as mere formatting changes. It could lead to unnecessary conflicts between different commits.

But since this part of the schema is only used for the delis toolspecifics you can leave it like this.